ttrpc-codegen/src/parser.rs

```diff
@@ -2182,7 +2182,7 @@ mod test {
         "#;
 
         let err = FileDescriptor::parse(msg).err().expect("err");
-        assert_eq!(4, err.line);
+        assert_eq!(3, err.line);
```

Comment:
@Tim-Zhang, @lifupan - I've made this change to get the tests to pass, however, please can you check this is a valid change. Looking at the history, there have been a few fd leak fixes which might explain why this change is needed?

Reply:
It's a valid change.

Reply:
The unit test has never passed before, thank you